collectors/cgroups.plugin/cgroup-name.sh.in

```diff
@@ -64,6 +64,22 @@ function docker_get_name_api() {
 	return 0
 }
 
+function k8s_get_name() {
+	local id="${1}"
+	KUBE_TOKEN="$(</var/run/secrets/kubernetes.io/serviceaccount/token)"
+	NAME=$(
+curl -sSk -H "Authorization: Bearer $KUBE_TOKEN"  "https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_PORT_443_TCP_PORT/api/v1/pods" |
```

Comment:
This is a "happy path" scenario, but you need to start somewhere :)

Reply:
why happy path? which failure conditions do you expect?

Reply:
RBAC limitations to k8s API

Reply:
We'll be setting the correct RBAC in the helm chart, but we should be checking for access to the API.

Reply:
NAME doesn't get set if anything fails and the case is handled by the code that was added to the same function, so this is ok now.